sonata-project/page-bundle/3.9/config/routes/sonata_page.yaml

```diff
@@ -0,0 +1,11 @@
+#sonata_page_api:
+#    resource: '@SonataPageBundle/Resources/config/routing/api.xml'
+#    prefix: /
+
+#sonata_page_cache:
```

Comment:
Why are these routes in comments?

Reply:
These routes are optional, because api and cache routes will not work out from the box. Other bundles must be installed and/or configured to make them work

Reply:
Hm.. okey. I usually prefer these routes to be added in an other recipe. 
But I know the sonata bundles "works really well together" or "have a strong coupling". It may be better for the user experience if these were left as they are.